src/coreComponents/physicsSolvers/contact/LagrangianContactSolver.cpp

```diff
@@ -1068,7 +1065,7 @@ void LagrangianContactSolver::
 
             switch( fractureState[kfe] )
             {
-              case FractureState::Stick:
+              case extrinsicMeshData::contact::FractureState::Stick:
```

Comment:
If it works, maybe something like `using namespace extrinsicMeshData::contact;`?

Reply:
yeah, I don't understand why sometimes it doesn't work. I think that, in this function it was not working...

Reply:
fatalitas!